src/providers/fault-count/cat-d/fault-count.cat-d.ts

```diff
@@ -210,7 +222,7 @@ export class FaultCountDHelper {
 
     const seriousFaultSumOfSimpleCompetencies = Object.keys(pickBy(seriousFaults)).length;
     const vehicleCheckSeriousFaults =
-      vehicleChecks ? FaultCountDHelper.getVehicleChecksFaultCountCatD(vehicleChecks).seriousFaults : 0;
+      vehicleChecks ? FaultCountDHelper.getVehicleChecksFaultCountNonTrailer(vehicleChecks).seriousFaults : 0;
```

Comment:
should be:

`this.getVehicleChecksFaultCountNonTrailer(vehicleChecks).seriousFaults`